test/unit/network_spec.js

```diff
@@ -176,4 +176,62 @@ describe('network', function() {
       done.fail(reason);
     });
   });
+
+  describe('PDFNetworkStreamFullRequestReader_extractNameFrom', function () {
```

Comment:
`describe('header filename extraction', ...` is a bit shorter and easier to read here.